lib/trento_web/plugs/api_redirector.ex

```diff
@@ -23,20 +23,23 @@ defmodule TrentoWeb.Plugs.ApiRedirector do
   import Plug.Conn
 
   @impl true
-  def init(opts), do: opts
+  def init(opts) do
+    available_api_versions =
+      Keyword.get(opts, :available_api_versions) ||
+        raise ArgumentError, "expected :available_api_versions option"
 
-  @impl true
-  def call(%Plug.Conn{path_info: [_ | path_parts], method: method} = conn, opts) do
-    available_api_versions = Keyword.get(opts, :available_api_versions)
-    router = Keyword.get(opts, :router)
+    if Enum.empty?(available_api_versions),
+      do: raise(ArgumentError, ":available_api_versions must have 1 element at least")
 
-    unless available_api_versions do
-      raise ArgumentError, "expected :available_api_versions option"
-    end
+    opts[:router] || raise ArgumentError, "expected :router option"
```

Comment:
Why we don't use `Keyword.get` for this option parameter? Maybe we could align the two methods of retrieving the options

Reply:
Alright! I will use `Keyword.get` in all of them and merge.
Some of the `Plug` examples I saw don't use it, but let's have our code using same patterns